lnrpc/walletrpc/walletkit_server.go

```diff
@@ -445,6 +449,26 @@ func (w *WalletKit) NextAddr(ctx context.Context,
 	}, nil
 }
 
+// ListWalletAddresses returns a list of all addresses known to the underlying wallet.
+func (w *WalletKit) ListWalletAddresses(ctx context.Context,
+	req *AddrRequest) (*AddrsResponse, error) {
+
+	var addrsSlice []string
+
+	walletAddrs, err := w.cfg.Wallet.ListWalletAddresses(lnwallet.WitnessPubKey)
```

Comment:
Why only `WitnessPubkey` here?

Reply:
@carlaKC I based that implementation detail off of the `NextAddress` command -- since that RPC only returns a WPK, it made sense to me to have this only return WPKs as well.